File: 0058-length-of-last-word/0058-length-of-last-word.java
class Solution {
    public int lengthOfLastWord(String s) {
        int count = 0;
        String str = s.trim();
        for(int i=str.length()-1;i>=0;i--){
            if(str.charAt(i) != ' '){
                count++;
            }else{
                break;
            }
        }
        return count;
    }
}

File: 0058-length-of-last-word/0058-length-of-last-word.js
/**
 * @param {string} s
 * @return {number}
 */
var lengthOfLastWord = function(s) {
    const newS = s.trim();
    let count = 0;
    for(let i=newS.length-1;i>=0;i--){
        if(newS.charAt(i) != ' '){
            count++;
        }else{
            break;
        }
    }
    return count;
};